helpers/configHelpers.js

```diff
@@ -32,6 +32,34 @@ function deleteRemovedKeys(dc, cc) {
   }
 }
 
+/**
+ * takes two (partial) webpack plugin configs and compares if a key was overwritten in custom config to then merge the otions to one plugin initialization
+
+ * @param {object} dc default configuration in current environment
+ * @param {object} cc custom configuration from project in current environment
+ * @returns {object, object} Default an Custom Configs with priority to custom config
+ */
+function mergeConfigPlugins(dc, cc) {
```

Comment:
I prefer speaking names for both arguments and variables.

Reply:
Changed that